2015-11-02 18:24:00+0900 [-] Log opened.
2015-11-02 18:24:00+0900 [-] twistd 15.4.0 (/usr/bin/python 2.7.6) starting up.
2015-11-02 18:24:00+0900 [-] reactor class: twisted.internet.epollreactor.EPollReactor.
2015-11-02 18:24:00+0900 [-] Starting BuildMaster -- buildbot.version: 0.8.12
2015-11-02 18:24:00+0900 [-] Loading configuration from '/home/yokotar/sandbox/buildbot/master.cfg'
2015-11-02 18:24:00+0900 [-] Setting up database with URL 'sqlite:///state.sqlite'
2015-11-02 18:24:00+0900 [-] setting database journal mode to 'wal'
2015-11-02 18:24:00+0900 [-] adding 1 new changesources, removing 0
2015-11-02 18:24:00+0900 [-] gitpoller: using workdir '/home/yokotar/sandbox/buildbot/gitpoller-workdir'
2015-11-02 18:24:00+0900 [-] adding 1 new slaves, removing 0
2015-11-02 18:24:00+0900 [-] adding 1 new builders, removing 0
2015-11-02 18:24:00+0900 [-] trying to load status pickle from /home/yokotar/sandbox/buildbot/runtests/builder
2015-11-02 18:24:00+0900 [-] no saved status pickle, creating a new one
2015-11-02 18:24:00+0900 [-] added builder runtests with tags None
2015-11-02 18:24:00+0900 [-] PBServerFactory starting on 9988
2015-11-02 18:24:00+0900 [-] Starting factory <twisted.spread.pb.PBServerFactory instance at 0x7f3419701440>
2015-11-02 18:24:00+0900 [-] adding scheduler 'all'
2015-11-02 18:24:01+0900 [-] adding scheduler 'force'
2015-11-02 18:24:01+0900 [-] WebStatus using (/home/yokotar/sandbox/buildbot/public_html)
2015-11-02 18:24:01+0900 [-] RotateLogSite starting on 8011
2015-11-02 18:24:01+0900 [-] Starting factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f34186b4248>
2015-11-02 18:24:01+0900 [-] Setting up http.log rotating 10 files of 10000000 bytes each
2015-11-02 18:24:01+0900 [-] BuildMaster is running
2015-11-02 18:25:11+0900 [-] beginning configuration update
2015-11-02 18:25:11+0900 [-] Loading configuration from '/home/yokotar/sandbox/buildbot/master.cfg'
2015-11-02 18:25:11+0900 [-] adding 1 new changesources, removing 1
2015-11-02 18:25:11+0900 [-] gitpoller: using workdir '/home/yokotar/sandbox/buildbot/gitpoller-workdir'
2015-11-02 18:25:11+0900 [-] removing scheduler 'force'
2015-11-02 18:25:11+0900 [-] adding scheduler 'force'
2015-11-02 18:25:11+0900 [-] (TCP Port 8011 Closed)
2015-11-02 18:25:11+0900 [-] Stopping factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f34186b4248>
2015-11-02 18:25:11+0900 [-] WebStatus using (/home/yokotar/sandbox/buildbot/public_html)
2015-11-02 18:25:11+0900 [-] RotateLogSite starting on 8011
2015-11-02 18:25:11+0900 [-] Starting factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f341869d680>
2015-11-02 18:25:11+0900 [-] Setting up http.log rotating 10 files of 10000000 bytes each
2015-11-02 18:25:11+0900 [-] configuration update complete
2015-11-02 18:28:22+0900 [-] beginning configuration update
2015-11-02 18:28:22+0900 [-] Loading configuration from '/home/yokotar/sandbox/buildbot/master.cfg'
2015-11-02 18:28:22+0900 [-] adding 1 new changesources, removing 1
2015-11-02 18:28:22+0900 [-] gitpoller: using workdir '/home/yokotar/sandbox/buildbot/gitpoller-workdir'
2015-11-02 18:28:22+0900 [-] removing scheduler 'force'
2015-11-02 18:28:22+0900 [-] adding scheduler 'force'
2015-11-02 18:28:22+0900 [-] (TCP Port 8011 Closed)
2015-11-02 18:28:22+0900 [-] Stopping factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f341869d680>
2015-11-02 18:28:22+0900 [-] WebStatus using (/home/yokotar/sandbox/buildbot/public_html)
2015-11-02 18:28:22+0900 [-] RotateLogSite starting on 8011
2015-11-02 18:28:22+0900 [-] Starting factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f34186cf5a8>
2015-11-02 18:28:22+0900 [-] Setting up http.log rotating 10 files of 10000000 bytes each
2015-11-02 18:28:22+0900 [-] configuration update complete
2015-11-02 18:31:51+0900 [-] beginning configuration update
2015-11-02 18:31:51+0900 [-] Loading configuration from '/home/yokotar/sandbox/buildbot/master.cfg'
2015-11-02 18:31:51+0900 [-] adding 1 new changesources, removing 1
2015-11-02 18:31:51+0900 [-] gitpoller: using workdir '/home/yokotar/sandbox/buildbot/gitpoller-workdir'
2015-11-02 18:31:51+0900 [-] removing scheduler 'force'
2015-11-02 18:31:51+0900 [-] adding scheduler 'force'
2015-11-02 18:31:51+0900 [-] (TCP Port 8011 Closed)
2015-11-02 18:31:51+0900 [-] Stopping factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f34186cf5a8>
2015-11-02 18:31:51+0900 [-] WebStatus using (/home/yokotar/sandbox/buildbot/public_html)
2015-11-02 18:31:51+0900 [-] RotateLogSite starting on 8011
2015-11-02 18:31:51+0900 [-] Starting factory <buildbot.status.web.baseweb.RotateLogSite instance at 0x7f341864bcb0>
2015-11-02 18:31:51+0900 [-] Setting up http.log rotating 10 files of 10000000 bytes each
2015-11-02 18:31:51+0900 [-] configuration update complete
2015-11-02 18:36:15+0900 [-] while polling for changes
	Traceback (most recent call last):
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 393, in callback
	    self._startRunCallbacks(result)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 501, in _startRunCallbacks
	    self._runCallbacks()
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 1184, in gotResult
	    _inlineCallbacks(r, g, deferred)
	--- <exception caught here> ---
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 1126, in _inlineCallbacks
	    result = result.throwExceptionIntoGenerator(g)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
	    return g.throw(self.type, self.value, self.tb)
	  File "/usr/local/lib/python2.7/dist-packages/buildbot-0.8.12-py2.7.egg/buildbot/changes/gitpoller.py", line 162, in poll
	    [self.repourl] + refspecs, path=self.workdir)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/buildbot-0.8.12-py2.7.egg/buildbot/changes/gitpoller.py", line 309, in _convert_nonzero_to_failure
	    % (command, args, path, self.repourl, code, stderr))
	exceptions.EnvironmentError: command fetch ['git://bitbucket.org/rioyokota/sandbox.git', '+master:refs/buildbot/git%3A%2F%2Fbitbucket.org%2Frioyokota%2Fsandbox.git/master'] in /home/yokotar/sandbox/buildbot/gitpoller-workdir on repourl git://bitbucket.org/rioyokota/sandbox.git failed with exit code 128: fatal: unable to connect to bitbucket.org:
	bitbucket.org[0: 131.103.20.167]: errno=Connection timed out
	bitbucket.org[1: 131.103.20.168]: errno=Connection timed out
	
	
	
2015-11-02 18:40:35+0900 [-] while polling for changes
	Traceback (most recent call last):
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 393, in callback
	    self._startRunCallbacks(result)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 501, in _startRunCallbacks
	    self._runCallbacks()
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 1184, in gotResult
	    _inlineCallbacks(r, g, deferred)
	--- <exception caught here> ---
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 1126, in _inlineCallbacks
	    result = result.throwExceptionIntoGenerator(g)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
	    return g.throw(self.type, self.value, self.tb)
	  File "/usr/local/lib/python2.7/dist-packages/buildbot-0.8.12-py2.7.egg/buildbot/changes/gitpoller.py", line 162, in poll
	    [self.repourl] + refspecs, path=self.workdir)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/buildbot-0.8.12-py2.7.egg/buildbot/changes/gitpoller.py", line 309, in _convert_nonzero_to_failure
	    % (command, args, path, self.repourl, code, stderr))
	exceptions.EnvironmentError: command fetch ['git://bitbucket.org/rioyokota/sandbox.git', '+master:refs/buildbot/git%3A%2F%2Fbitbucket.org%2Frioyokota%2Fsandbox.git/master'] in /home/yokotar/sandbox/buildbot/gitpoller-workdir on repourl git://bitbucket.org/rioyokota/sandbox.git failed with exit code 128: fatal: unable to connect to bitbucket.org:
	bitbucket.org[0: 131.103.20.167]: errno=Connection timed out
	bitbucket.org[1: 131.103.20.168]: errno=Connection timed out
	
	
	
2015-11-02 18:43:39+0900 [Broker,0,127.0.0.1] slave 'example-slave' attaching from IPv4Address(TCP, '127.0.0.1', 34452)
2015-11-02 18:43:39+0900 [Broker,0,127.0.0.1] Got slaveinfo from 'example-slave'
2015-11-02 18:43:39+0900 [Broker,0,127.0.0.1] Starting buildslave keepalive timer for 'example-slave'
2015-11-02 18:43:39+0900 [Broker,0,127.0.0.1] bot attached
2015-11-02 18:43:39+0900 [Broker,0,127.0.0.1] Buildslave example-slave attached to runtests
2015-11-02 18:44:55+0900 [-] while polling for changes
	Traceback (most recent call last):
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 393, in callback
	    self._startRunCallbacks(result)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 501, in _startRunCallbacks
	    self._runCallbacks()
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 1184, in gotResult
	    _inlineCallbacks(r, g, deferred)
	--- <exception caught here> ---
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 1126, in _inlineCallbacks
	    result = result.throwExceptionIntoGenerator(g)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/python/failure.py", line 389, in throwExceptionIntoGenerator
	    return g.throw(self.type, self.value, self.tb)
	  File "/usr/local/lib/python2.7/dist-packages/buildbot-0.8.12-py2.7.egg/buildbot/changes/gitpoller.py", line 162, in poll
	    [self.repourl] + refspecs, path=self.workdir)
	  File "/usr/local/lib/python2.7/dist-packages/Twisted-15.4.0-py2.7-linux-x86_64.egg/twisted/internet/defer.py", line 588, in _runCallbacks
	    current.result = callback(current.result, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/buildbot-0.8.12-py2.7.egg/buildbot/changes/gitpoller.py", line 309, in _convert_nonzero_to_failure
	    % (command, args, path, self.repourl, code, stderr))
	exceptions.EnvironmentError: command fetch ['git://bitbucket.org/rioyokota/sandbox.git', '+master:refs/buildbot/git%3A%2F%2Fbitbucket.org%2Frioyokota%2Fsandbox.git/master'] in /home/yokotar/sandbox/buildbot/gitpoller-workdir on repourl git://bitbucket.org/rioyokota/sandbox.git failed with exit code 128: fatal: unable to connect to bitbucket.org:
	bitbucket.org[0: 131.103.20.168]: errno=Connection timed out
	bitbucket.org[1: 131.103.20.167]: errno=Connection timed out
	
	
	
